### DISCUSSION TITLE
Cron Jobs

### DISCUSSION THREAD

**Discussion by brewhousedigital** [pocketbase/pocketbase#862] [Q&A] [answered]
Does PocketBase support Cron Jobs? If I wanted to loop through a table every night and check for specific values and then act on those rows, is that possible?

**Comment by ganigeorgiev** [comment, marked as answer]
You have 2 options if you use PocketBase as framework:

1. Register custom command and use the os utilities like `crontab` to call it.
    The documentation currently is a little lacking on this, but here is an example how to register a custom command (internally we are using [`spf13/cobra`](https://github.com/spf13/cobra)):
    ```go
    app.RootCmd.AddCommand(&cobra.Command{
        Use: "hello",
        Run: func(command *cobra.Command, args []string) {
            log.Println("Hello world!")
        },
    })
    ```

2. Use a Go job-scheduling library like [`robfig/cron`](https://github.com/robfig/cron), [`go-co-op/gocron`](https://github.com/go-co-op/gocron), etc.
